### PR TITLE
Fix inconsistent copyright for C# files in Mono's corelib

### LINKED ISSUE #34991: Inconsistent copyright headers in Mono
The C# files in Mono's corelib have inconsistent (or completely missing) copyright headers. Sample:

```
// Copyright 2016 Dot net foundation.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

### PR DESCRIPTION
**Pull request by fanyang-mono**
Fixes #34991

### CODE REVIEW

File: src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeMarshal.cs [hunk 1]
```diff
@@ -1,3 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```
Comment (stephentoub) [inline src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeMarshal.cs:3]:
```suggestion
// See the LICENSE file in the project root for more information.

```

File: src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs [hunk 2]
```diff
@@ -1,12 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```
Comment (stephentoub) [inline src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:2]: In the past, we'd been leaving out this middle line when adding the license header to files from Mono.
@richlander, can you comment on what we should be doing moving forward here?

File: src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+// See the LICENSE file in the project root for more information.
 //
 // Mono runtime native structs surfaced to managed code.
 //
 // Authors:
```
Comment (SamMonoRT) [inline src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:7]: @marek-safar - while we are updating headers, does it make sense to get rid of the authors of the file (has xamarin or ximian email ids) ?(This if for files we are updating here)

Comment (marek-safar) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:7]: We could if it makes the process easier

Comment (fanyang-mono) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:7]: I was wondering about that too, then I saw library also has files like this. For example, https://github.com/dotnet/runtime/blob/master/src/libraries/System.Drawing.Common/src/System/Drawing/Bitmap.Unix.cs

But if our ultimate goal is to get rid of those, I could do have my script doing that as well.

Comment (SamMonoRT) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:7]: Looking at a few coreclr files, I would suggest removing it for the files you are touching here

File: src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs (hunk 2 above)
Comment (richlander) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:2]: Either the two line or three line version is fine. And yes, its the middle line that can be skipped.

File: src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs (hunk 3 above)
Comment (SamMonoRT) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:7]: we'll create a separate tracking issue to remove author names. Resolving this comment

File: src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs (hunk 2 above)
Comment (SamMonoRT) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:2]: We'll leave the 3 lines in there. Resolving comment.

**Comment by SamMonoRT** [comment]
LGTM - @stephentoub any concerns approving and merging this ?

**Review by SamMonoRT — APPROVED** [review]

File: src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs (hunk 2 above)
Comment (richlander) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:2]: Correction ... This is the 2-line version we decided on:

```csharp
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license
```

Comment (SamMonoRT) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:2]: @fanyang-mono - please update to the 2-line version. Thank you.

Comment (fanyang-mono) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:2]: Rich said that either two line or three line version is fine. Why do we have to use the two line version?

Comment (fanyang-mono) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:2]: @richlander Just to clarify, the 3-line version is fine as well, right?

Comment (stephentoub) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:2]: I'm currently in the process of trying to move the entire repo to the 2-line version.

Comment (fanyang-mono) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:2]: I see. I will make my change into 2-line version then.

Comment (fanyang-mono) [reply src/mono/netcore/System.Private.CoreLib/src/Mono/RuntimeStructs.cs:2]: @stephentoub I updated the copyright to 2-line version. Could you please review?

**Review by stephentoub — APPROVED** [review]

**Comment by fanyang-mono** [comment]
Check passed and reviewed. Ready to merge.

**Pull request merged**